website_portal_contract/demo/account_analytic_account_demo.xml

```diff
@@ -13,8 +13,6 @@
         <field name="partner_id" ref="portal.demo_user0_res_partner" />
         <field name="recurring_invoices" eval="True" />
         <field name="website_template_id" ref="website_contract_template_default" />
-        <field name="recurring_invoice_line_ids"
```

Comment:
Do you remove this?

Reply:
Yup, [it's a O2M](https://github.com/OCA/contract/blob/10.0/contract/models/account_analytic_account.py#L26) so it is set automatically when the line is set to the analytic account.

Reply:
Yeah, totally! It's important then the order of the XML loading in the manifest. Note that in it for avoiding problems changing that order.

Reply:
Added a note in the manifest about the load order. 

Note that I didn't increase the version of the module because this is all just demo data issue.